docker-compose.yml

```diff
@@ -121,6 +126,7 @@ services:
       traefik.frontend.rule: PathPrefix:/api/resource-manager
   deployer:
     image: ${IMAGE_REPOSITORY}renga-deployer:$PLATFORM_VERSION
+    command: uwsgi /code/docker/uwsgi/uwsgi.ini
```

Comment:
Makes deployer fail on startup, overkill and prevents stack traces from being printed.